saleor/graphql/checkout/types.py

```diff
@@ -176,6 +180,19 @@ def resolve_available_shipping_methods(root: models.Checkout, info):
         )
         if available is None:
             return []
+
+        if not include_taxes_in_prices() and display_gross_prices():
```

Comment:
1. Why do you validate `include_taxes_in_prices` here?
All tax responsibilities should be transferred to the tax plugin. 
2. IMO it is not needed. Based on Vatlayer - `manager.apply_taxes_to_shipping` calls plugin's apply_taxes_to_shipping -> `get_taxed_shipping_price` -> `apply_tax_to_price` and inside this method plugin checks `include_taxes_in_prices`

Reply:
price field is MoneyField not TaxMoneyField that's why I thought that I should distinguish net and gross value
to do that I need to know which type is in DB (gross or net) and should I display gross or net.

Reply:
Yes, you should check that, but there is no need to check `include_taxes...`. See `saleor.checkout.forms.ShippingMethodChoiceField#label_from_instance`.
Plugin should handle `include_taxes_in_prices` (as Vatlayer do).